userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,124 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect from as few as one example per category. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild. It is best suited for 2D objects or when there isn't much expected variance in the examples, such as playing cards, logos, road signs and clapperboards. One-Shot detector is not suitable for 3D objects such as faces, animals and cars. 
+
+Given an image and a one-shot detector trained on category **stop sign**, the output prediction will look like:
```

Comment:
This sentence seems a bit odd and out of place. Is this supposed to be an example intro here. If so, it might need some more context.

Reply:
The purpose of this sentence was to give users a glimpse into what the end product of the toolkit looks like (by showing the prediction image under it), before diving further into how to train a model. I do see what you mean by it seeming out of place. I'm thinking of a better way of presenting this.